src/sim/trace-extract/google_trace_simulator.cc

```diff
@@ -774,7 +1022,42 @@ void GoogleTraceSimulator::ResetUuidAndAddResource(
                                               GetCurrentTimestamp())));
 }
 
-void GoogleTraceSimulator::ReplayTrace() {
+const std::string CHANGE_STATS_HEADER = "total_changes,new_node,remove_node,new_arc,change_arc,remove_arc";
+void output_change_stats(DIMACSChangeStats &stats, ofstream &stats_file) {
+	stats_file << stats.total << "," << stats.new_node << ","
+			       << stats.remove_node << "," << stats.new_arc << ","
+						 << stats.change_arc << "," << stats.remove_arc
+						 << std::endl;
+}
+
+int get_binary_directory(char *pBuf, ssize_t len) {
```

Comment:
Moved to `misc/utils.cc`, since this is more general functionality that will be useful outside the simulator.